src/reducer/notesReducer.js

```diff
@@ -0,0 +1,19 @@
+export const notesReducer = (state, action) => {
+  switch (action.type) {
+    case "ADD_NOTES":
```

Comment:
Suggestion: can use variables instead of strings to avoid typos.

Reply:
Thank for pointing it out, Made the changes